ejercicio1/multimedia.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Multimedia</title>
+</head>
+<body>
+<section>
+	<h1>Multimedia</h1>
+		<h3>Imagen</h3>
+			<img src="assets/img/videl.jpg">
```

Comment:
What happen with the accessibility in this case? What happen if the image don't load?